.gitallowed

```diff
@@ -1,8 +1,9 @@
-.*git-secrets/.*:*
-.*terraform.tfstate.*:*
-.*.svg:*
-.*.jquery.*.js:*
-.*engine_version*
-.*assembly: AssemblyVersion*
-.*assembly: AssemblyFileVersion*
-.*Version=*
+.*git-secrets/.*:*
+.*terraform.tfstate.*:*
+.*.svg:*
+.*.jquery.*.js:*
+.*engine_version*
+.*assembly: AssemblyVersion*
+.*assembly: AssemblyFileVersion*
+.*Version=*
+0.0.0.0
```

Comment:
By default the `git-secrets` script flags any IP address as a secret, `0.0.0.0` is a non-routable address used to designate an invalid, unknown or non-applicable target - so not _really_ a secret. 

We use this IP to run `uvicorn` locally in the sandbox mock receiver, so it'll likely be deprecated with the move over to the AWS mock receiver anyway.